bigbluebutton-client/resources/prod/lib/kurento-extension.js

```diff
@@ -28,6 +29,8 @@ Kurento = function (
   this.screen = null;
   this.webRtcPeer = null;
   this.mediaCallback = null;
+  this.started = false;
+  this.reconnectionAttemps = 0;
```

Comment:
The var name should be "reconnectionAttempts". Also, will it be reset if someone stops screensharing and then starts it again?

Reply:
It will. The whole `Kurento` object is deleted when screensharing stops, and so is that variable.